lib/commands/element.js

```diff
@@ -40,6 +40,26 @@ commands.getText = async function (elementId) {
   return await this.uiautomator2.jwproxy.command(`/element/${elementId}/text`, 'GET', {});
 };
 
+commands.click = async function (elementId) {
+  return await this.uiautomator2.jwproxy.command(`/element/${elementId}/click`, 'POST', {elementId});
+};
+
+commands.tap = async function (elementId, x = 0, y = 0, count = 1) {
+  for (let i = 0; i < count; i++) {
+    if (elementId) {
+      // we are either tapping on the default location of the element
+      // or an offset from the top left corner
+      if (x !== 0 || y !== 0) {
+        await this.uiautomator2.jwproxy.command(`/element/${elementId}/click`, 'POST', {elementId, x, y});
+      } else {
+        await this.uiautomator2.jwproxy.command(`/element/${elementId}/click`, 'POST', {elementId});
+      }
+    } else {
+      await this.uiautomator2.jwproxy.command(`/element/${elementId}/click`, 'POST', {x, y});
```

Comment:
In this block there is no `elementId` (or, at least, it is falsy), so what is put into the endpoint of the call? It could be any number of things, none of which make much sense (e.g., `'/element/undefined/click'`).